eng/common/scripts/Delete-RemoteBranches.ps1

```diff
@@ -5,6 +5,10 @@ param(
   $RepoName,
   # Please use the RepoOwner/RepoName format: e.g. Azure/azure-sdk-for-java
   $RepoId="$RepoOwner/$RepoName",
+  # CentralRepoId the original PR to generate sync PR. E.g Azure/azure-sdk-tools for eng/common
+  $CentralRepoId,
+  # We start from the sync PRs, use the branch name to get the PR number of central repo. E.g. sync-eng/common-(<branchName>)-(<PrNumber>). Have group name on PR number.
+  $CentralPRRegex,
```

Comment:
We should change BranchPrefix to BranchRegex as there isn't any real reason to have duplication.

Reply:
Lets rename this to BranchRegex so it is clear that is for both purposes.